app/models/manageiq/providers/hawkular/middleware_manager.rb

```diff
@@ -1,3 +1,5 @@
+require 'hawkular/hawkular_client_utils'
```

Comment:
On a previous iteration, the underlying Hawkular gem wasn't including this but a code down the stack was trying to use a method defined on the mixin. So, this was kinda like a hot fix for a problem not related to this work.

The gem has been fixed, so, there's no reason to have this code anymore.